lib/matplotlib/dviread.py

```diff
@@ -1013,17 +1013,23 @@ def find_tex_file(filename, format=None):
     if isinstance(format, bytes):
         format = format.decode('utf-8', errors='replace')
 
+    if os.name == 'nt':
+        # On Windows only, kpathsea can use utf-8 for cmd args and output.
+        # The `command_line_encoding` environment variable is set to force it
+        # to always use utf-8 encoding. See mpl issue #11848 for more info.
+        kwargs = dict(env=dict(os.environ, command_line_encoding='utf-8'))
+    else:
+        kwargs = {}
+
     cmd = ['kpsewhich']
     if format is not None:
         cmd += ['--format=' + format]
     cmd += [filename]
     try:  # Below: strip final newline.
-        result = cbook._check_and_log_subprocess(cmd, _log)[:-1]
```

Comment:
#12356 and this differ here....  Can that PR and this one be combined?

Reply:
@Kojoley Can you include the fix from #12356 into this PR?

Reply:
Yes, should I cherry pick your commit into my PR?

Reply:
cherry-pick or just reproduce it, either way is fine.